contrib/postgres_proxy/filters/network/source/postgres_filter.cc

```diff
@@ -211,7 +211,7 @@ bool PostgresFilter::onSSLRequest() {
   buf.add("S");
   // Add callback to be notified when the reply message has been
   // transmitted.
-  read_callbacks_->connection().addBytesSentCallback([=](uint64_t bytes) -> bool {
+  read_callbacks_->connection().addBytesSentCallback([=, this](uint64_t bytes) -> bool {
```

Comment:
I've been using lambdas for a long time but had never seen an `=` capture before! I don't see what benefit this change provides, based on the C++ doc (https://en.cppreference.com/w/cpp/language/lambda):
```
    [=, this] {};  // until C++20: Error: this when = is the default
                   // since C++20: OK, same as [=]
```

Reply:
I'm not an export either on c++ or lambdas, I raise this up because istio/proxy enable this filter by default. The build error appears recently(I believe related to thes macos upgrade), the change is suggested by the complier.